### DISCUSSION TITLE
How to put TDM score into TAB GUI?

### DISCUSSION THREAD

**Discussion by joaoglibras** [piqueserver/piqueserver#739] [Q&A] [answered]
I made a TDM server, but it doesn't show each team kill score in TAB menu. 

The amount of kills needed to win is 100. But in TAB menu, it shows "0-10 vs 0-10", because it's using the intel cap limit as a counter, and only intel caps scores a point for the team.

Yeah, I've set **remove_intel** to **True**. So there is no Intels on the map.

Can't figure what to do. Thanks in advance, that's it :D

**Comment by utf-4096** [comment, marked as answer]
TDM is not an "official" gamemode in a way, so clients like voxlap/betterspades/openspades don't show the TDM score because the AoS protocol doesn't support it, unfortunately.

**Reply by NotAFile** [reply]
One way would be to send an Intel capture for every 10% of the goal, but nobody has added that yet

**Reply by joaoglibras** [reply]
> TDM is not an "official" gamemode in a way, so clients like voxlap/betterspades/openspades don't show the TDM score because the AoS protocol doesn't support it, unfortunately.

Thank you for the answer. I would spend hours trying to fix such a thing.
